metrics/web/test/auth/presentation/widgets/auth_form_test.dart

```diff
@@ -293,6 +310,110 @@ void main() {
         expect(tappableAreaFinder, findsOneWidget);
       },
     );
+
+    testWidgets(
+      "displays the email auth error message",
+      (tester) async {
+        final authNotifier = AuthNotifierMock();
+
+        when(authNotifier.isLoading).thenReturn(false);
+        when(authNotifier.emailErrorMessage).thenReturn(errorMessage);
```

Comment:
We should mock the notifier in widget tests instead of use cases. Could you, please, remove all occurrence of use case mocks?